src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -821,8 +834,10 @@ public function process_payment( $order_id ) {
 			);
 		}
 
+		// We ignore the nonce verification because process_payment is called after validate_fields.
```

Comment:
put it one line bellow, just before the effective `phpcs:ignore` comment

Reply:
Done